lib/normalizers/v1.js

```diff
@@ -242,7 +241,13 @@ _.assign(Builders.prototype, {
 
         auth = self.auth(requestV1);
 
-        if (auth) { requestV1.auth = auth; }
+        if (auth) {
+            requestV1.auth = auth;
+
+            if (_.has(requestV1, 'helperAttributes') && !requestV1.currentHelper) {
```

Comment:
Can you make this in a separate PR? Is this blocking this PR?

Reply:
Yes.